webpack.config.js

```diff
@@ -12,7 +12,7 @@ function resolve (dir) {
 
 const config = {
   entry: [
-    resolve('./components/index.js')
+    resolve('./index.js')
```

Comment:
i think there is no need to have the index file here. instead, i'd also move it into /src, and specify that here like 

```suggestion
    resolve('./src/index.js')
```

this way you can also drop the `./src/` prefixes in every import statement in the index.js.

Reply:
I think so, too. Done her [88792b4](https://github.com/demos-europe/demosplan-ui/pull/13/commits/88792b4c918516a7b14bacc779208738a24ed628)